analysis/paper.Rmd

```diff
@@ -38,9 +38,11 @@ The original model was implemented in R (v3.5.2) [team2013r] with web applicatio
 The goal of this large-scale simulation used in the original paper is to build a wide range of demographic histories that accounts for the variability between actual populations, to measure how biases such as the independence assumption of mother and children survival rate in a population influenced by HIV/AIDS would exist. @quattrochi2019 used stochastic simulation methods to generate a set of population with input parameters such as demographic features, mortality rate for U5M, maternal mortality rate, HIV prevalence, etc. The input simulation parameters vary over a large interval for different age groups across different populations. In each time step of one year, each woman in the population has some probabilities to give birth for those who are 15 to 49 years old, being infected with HIV, initiating ART (if HIV-positive), and dying given the simulation parameters. Through the aging process in the simulation population, children born with infected HIV mothers will have a probability of getting infected while all children will have a probability of mortality.     
 
 For the calculation of bias, we used the formula provided in the original paper. 
-$$ Relative bias = {\frac{IE_{survivors} - IE_{survivors&HIV deaths}}{IE_{survivors&HIV deaths}} $$        
-$$ Absolute bias = IE_{survivors} - IE_{survivors&HIV deaths}$$    
-According to [@quattrochi2019], ${IE_{survivors}$ was the estimates of U5M based on living women in 2010, and IE_{survivors&HIV deaths} was the estimates of U5M based on living women in 2010 and women died from HIV/AIDS before 2010 given the condition that they would have been 15-49 in 2010 if they had survived.    
+$$Relative bias = \frac{IE_{survivors} - IE_{survivors,HIV deaths}}{IE_{survivors,HIV deaths}}$$
+
+$$Absolute bias = IE_{survivors} - IE_{survivors,HIV deaths}$$
```

Comment:
Very minor suggestion, $$\mbox{Relative bias} = \frac{IE_{survivors} - IE_{survivors,HIV deaths}}{IE_{survivors,HIV deaths}}$$, this will create space between relative and bias. Looks good otherwise!